REFERENCES.md

```diff
@@ -35,7 +35,6 @@
 | nipype | https://doi.org/10.3389/fninf.2011.00013 https://doi.org/10.5281/zenodo.581704 | https://github.com/nipy/nipype/ |
 | convert3d | | https://sourceforge.net/projects/c3d/ |
 | **Graphics**
-| seaborn | https://doi.org/10.5281/zenodo.883859 | https://github.com/mwaskom/seaborn |
```

Comment:
We do still use it, just not as a direct import. https://github.com/poldracklab/niworkflows/search?utf8=%E2%9C%93&q=seaborn&type=

Reply:
I'll update the palettes in niworkflows and credit seaborn over there. Would that be okay?

Reply:
I'd rather do it on the package that people actually use.

Reply:
Okay, rolled back.